draft-ietf-quic-multipath.md

```diff
@@ -323,13 +323,27 @@ client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
 
-If validation succeed, the client can send non-probing, 1-RTT packets
-on the new paths.  In contrast with the specification in
+If validation succeed, the client can continue to use the path.
+If validation fails, the client MUST NOT use the path and can
+remove any status assiciated to the path initation attempt.
+{{Section 9.1 of QUIC-TRANSPORT}} introduces the concept of
+"probing" and "non-probing" frames. When the  multipath extension
+is negoatiated, the receiption of "non-probing"
+packet on a new path needs to be consider as an path initiation
+attempt that does not impact the path status of any existing
+path. Therefore, any frame can be send on a new path at any time
+as long as the congestion control limits are respected.
```

Comment:
```suggestion
as long as the congestion control limits are respected.
Specially, bundling the PATH_RESPONSE frame with an MP_ACK frames
can be beneficial.
```
Do we want to add that as well? Or maybe somewhere else in the text?

Reply:
This is a point worth pointing out, though it might be better to state at a place where we say that ACK for a path should be sent consistently from one path.